src/main/java/org/mastodon/mamut/MamutAppModel.java

```diff
@@ -157,4 +160,16 @@ public BranchGraphSynchronizer getBranchGraphSync()
 	{
 		return branchGraphSync;
 	}
+
+	public void close() {
+		closeListeners.list.forEach( CloseListener::close );
+	}
+
+	/**
+	 * Listeners that are notified when the Mastodon project is closed,
+	 * by for example closing the main window.
+	 */
+	public Listeners< CloseListener > closeListeners() {
```

Comment:
The method name may be misleading and could also be interpreted as in "close the listeners". The javadoc explains it, but nevertheless maybe `getCloseListeners()` would be less ambiguous?

Reply:
I renamed the method to projectCloseListeners()